src/data/ACTIONS/root/PCT.ts

```diff
@@ -88,6 +88,7 @@ export const PCT = ensureActions({
 		onGcd: true,
 		speedAttribute: Attribute.SPELL_SPEED,
 		castTime: 2300,
+		gcdRecast: 3300,
```

Comment:
I probably should've checked both draft PRs, but the note I left on the other one is still applicable. gcdRecast on an action with a cooldown causes it to get treated like Phlegma. Downstream code should treat cooldown as gcdRecast if gcdRecast is missing and onGcd is true. These changes to PCT's actions should be removed.

Reply:
reverted and updated logic in the new section